src/calculator.js

```diff
@@ -1,47 +1,35 @@
-exports._check = () => {
+exports._check = (num1,num2) => {
```

Comment:
Block must not be padded by blank lines.

###### See http://eslint.org/docs/rules/padded-blocks for details about this rule